src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawFigure {

```

Comment:
Think about why using DrawFigure name for this interface is a mistake. (You can ask in the chat in case of have no ideas)
```suggestion
public interface Drawable {
```